test/api_test.py

```diff
@@ -19,11 +19,18 @@
 import logging
 import unittest
 
+import googleapiclient.errors
+import mock
+import nose.tools
+
 import hyou.api
 
 import http_mocks
```

Comment:
Do you mean the `http_mocks` one? It's because the `test` directory is not a package, so we can't do a relative import. Or is is something else? 😕